frame/assistants/feemarket-s2s-transition/Cargo.toml

```diff
@@ -26,17 +26,17 @@ serde       = { version = "1", features = ["derive"] }
 
 codec                  = { package = "parity-scale-codec", version = "2" }
 
```

Comment:
We can remove feemarket-s2s-transition for this pull request and directly use feemarket-s2s crate for darwinia-crab project.
Related: https://github.com/darwinia-network/bridger/pull/448#discussion_r851845926